skietbaan_fe/src/reducers/postReducer.js

```diff
@@ -0,0 +1,28 @@
+import {FETCH_POSTS, NEW_POST} from '../actions/types';
+
+const initialState = {
+    items: [],
+    item: {}
+
+}
+
+export default function(state = initialState, action){
+    switch(action.type){
+        case FETCH_POSTS:
+        console.log('reducer');
```

Comment:
Do not leave console logs in code. Also if you are using console logs to debug, VSCode has an extension called 'Debugger for Chrome' which allows you to debug using breakpoints 
Check this for more details: https://code.visualstudio.com/docs/nodejs/reactjs-tutorial#_debugging-react

Reply:
Sorted. Kindly review. Advise noted.